js/yobit.js

```diff
@@ -834,7 +902,39 @@ module.exports = class yobit extends Exchange {
             request['pair'] = market['id'];
         }
         const response = await this.privatePostActiveOrders (this.extend (request, params));
-        const orders = this.safeValue (response, 'return', []);
+        //
+        //      {
+        //          "success":1,
+        //          "return": {
+        //              "1101103634006799": {
+        //                  "pair":"doge_usdt",
+        //                  "type":"buy",
+        //                  "amount":10,
+        //                  "rate":0.1,
+        //                  "timestamp_created":"1650034937",
+        //                  "status":0
+        //              },
+        //              "1101103634006738": {
+        //                  "pair":"doge_usdt",
+        //                  "type":"buy",
+        //                  "amount":10,
+        //                  "rate":0.1,
+        //                  "timestamp_created":"1650034932",
+        //                  "status":0
+        //              }
+        //          }
+        //      }
+        //
+        const rawOrders = this.safeValue (response, 'return', {});
+        const ids = Object.keys (rawOrders);
+        const orders = [];
+        // temporary work-around for PHP id-order bug in parseOrders in Exchange.js
```

Comment:
We don't need this workaround if you upgrade to the most recent version of CCXT (it should be fixed, as far as I understand). Or do you still have this issue after upgrading?